lib/models/repository-states/present.js

```diff
@@ -611,6 +620,13 @@ export default class Present extends State {
     });
   }
 
+  // Author information
+
+  getAuthors(options) {
+    // TODO: caching
```

Comment:
just for my own education: is this a TODO you intend to resolve before you merge the pr, as it's a work in progress?  Or a longer term "fix this someday" kind of todo?